stetho/src/main/java/com/facebook/stetho/inspector/elements/Document.java

```diff
@@ -140,6 +140,12 @@ public void setAttributesAsText(Object element, String text) {
     mDocumentProvider.setAttributesAsText(element, text);
   }
 
+  public void getElementStyle(Object element, StyleAccumulator styleInfoAccumulator) {
```

Comment:
We still need to talk about the organization of this API, re: my older comment about this being, essesntially, a utility method

For now, this method's signature is slightly off ... it should be plural (`getElementStyles`) and the parameter should have the word "info" removed